js/level3.js

```diff
@@ -98,15 +161,17 @@
 
 
 /*
-    Ok, so far so good. But what if we need only content of our 'h1' tag? We have
-    another property for it - .innerHTML
+    Ok, so far so good. But what if we want only the text from our 'h1' tag?
+    Page elements have another property for this: '.textContent'
+
     Example:
-    ourTwitter.innerHTML;
-    //we will get text '@NodeGirlsSydney'
+
+    ourTwitter.textContent;
+    // we will get text '@NodeGirlsSydney'
```

Comment:
`we will get text "Twitter: @NodeGirlsSydney @NodeGirlsMelb"` as we had updates in html